### PR TITLE
chore: add prod config

### PR DESCRIPTION
**Pull request by elseagle**

### CODE REVIEW

**Comment by elseagle** [comment]
@johnseekins the pipeline is failing because black and flake8 are conflicting for a lengthy line in terms of formatting, I don't want to take any action on this yet (extend ignores for the errors) but I wanted to bring your attention to this to advice on way forward 


cc: @bfossen-ce

File: settings.py [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
 import boto3
 
+# Get secret and access key
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+
 # Get db paasword from AWS SSM
-session = boto3.Session(profile_name="default")
-ssm = session.client("ssm")
+session = boto3.Session(
```
Comment (johnseekins) [inline settings.py:13]: What changed that we need a key pair instead of just a profile name?

Comment (elseagle) [reply settings.py:13]: The initial version was not deployed, worked locally. On the lambda instance the has these key pairs in its default environment so we can reference them.

Comment (elseagle) [reply settings.py:13]: Reference here: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

**Comment by elseagle** [comment]
> @johnseekins the pipeline is failing because black and flake8 are conflicting for a lengthy line in terms of formatting, I don't want to take any action on this yet (extend ignores for the errors) but I wanted to bring your attention to this to advice on way forward
> 
> cc: @bfossen-ce

This is resolved now. @johnseekins

**Review by bfossen-ce — APPROVED** [review]
LGTM!

**Review by bfossen-ce — APPROVED** [review]
LGTM

**Review by bfossen-ce — APPROVED** [review]

**Pull request merged**